### PR TITLE
[DX] Add phpVersion() method to RectorConfig + make use of new config class in test configs

### PR DESCRIPTION
**Pull request by TomasVotruba**
@samsonasik Hi Abdul, could you handle the same changes in the split packages? I think it's time to upgrade to simpler configs :)

### CODE REVIEW

**Review by samsonasik — CHANGES_REQUESTED** [review]
@TomasVotruba I will try, for `e2e` sub directories, the `ContainerConfigurator` usage should be kept, except the `e2e/use-rector-configurator` one, which specifically test it, so both are tested for both using `ContainerConfigurator` and `RectorConfig`

**Comment by TomasVotruba** [comment]
@samsonasik Good point, I'll keep one just in case. In the end we will move to `RectorConfig` completely.

File: src/Configuration/Option.php [hunk 1]
```diff
@@ -62,12 +62,6 @@ final class Option
      */
     public const IMPORT_SHORT_CLASSES = 'import_short_classes';
 
-    /**
-     * @deprecated Narrow too complex import settings to more practical use.
-     * @var string
-     */
-    public const IMPORT_DOC_BLOCKS = 'import_doc_blocks';
-
```
Comment (alexander-schranz) [inline src/Configuration/Option.php:70]: This does currently crash our CI's. Is this expected to be removed in a patch version release? @TomasVotruba

Comment (TomasVotruba) [reply src/Configuration/Option.php:70]: I'm sorry about that. This option is no longer supported, so you can remove it.
Rector is still in dev 0.x, where breaks can happen.

Comment (alexander-schranz) [reply src/Configuration/Option.php:70]: Thx for the fast reponse. We will remove it from our configuration.

**Pull request merged**